rero/sonar
# deposit: improve form in submission process

* Removes `classification` from the required fields.
* Adds the possibility to fill a full date for document date field.
* Renames `title` property for publication.
* Removes `pages` from required fields in publication.
* Hides or shows publication depending on document type.
* Adds a placeholder for `pages` property in publications.
* Adds a placeholder for document date.
* Removes `affiliation` from required fields in contributors.
* Closes #rero/sonar-ui#62.
* Closes #rero/sonar-ui#68.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

## How to test
1. Go to Administration > Deposit a publication
2. Fill the form and check placeholders, constraints.
3. Change document type and check "Publication" fields are shown/hidden.
